test/e2e/specs/specs-wpcom/wp-editor__privacy-password.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * @group gutenberg
+ */
+
+import {
+	DataHelper,
+	GutenbergEditorPage,
+	setupHooks,
+	EditorSettingsSidebarComponent,
+	PostVisibilityOptions,
+	TestAccount,
+	PublishedPostPage,
+	BrowserHelper,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+describe( DataHelper.createSuiteTitle( `Editor: Post Privacy` ), function () {
+	const accountName = BrowserHelper.targetGutenbergEdge()
+		? 'gutenbergSimpleSiteEdgeUser'
+		: 'simpleSitePersonalPlanUser';
+	const content = DataHelper.getRandomPhrase();
+	const visibility = 'Password';
+	const postPassword = 'car';
+
+	let page: Page;
+	let url: string;
+	let gutenbergEditorPage: GutenbergEditorPage;
+	let editorSettingsSidebarComponent: EditorSettingsSidebarComponent;
+
+	setupHooks( async ( args ) => {
+		page = args.page;
+	} );
+
+	describe( `Create a Password protected post`, function () {
+		beforeAll( async function () {
+			const testAccount = new TestAccount( accountName );
+			await testAccount.authenticate( page );
+		} );
+
+		it( 'Go to the new post page', async function () {
+			gutenbergEditorPage = new GutenbergEditorPage( page );
+			await gutenbergEditorPage.visit( 'post' );
```

Comment:
When I was mapping out these specs, one of the goals I had was to try to create coverage for both pages and posts by having some of the specs use posts, and others pages.

In my original test plan map, I had slotted these privacy tests to run against pages, since many of the other ones already target posts. I also imagined that people might use the privacy feature more often on pages than posts (e.g. Admin pages).

What do you think of switching over these tests to use pages? Definitely not required, just thought I would throw out the idea!

Reply:
Sounds good - I will rework these into pages.

Reply:
Moved these specs into using `page`.